cmd/flags.go

```diff
@@ -101,7 +100,6 @@ var (
 	BootstrapNode = &cli.StringSliceFlag{
 		Name:  "bootstrap-node",
 		Usage: "The address of bootstrap node. Beacon node will connect for peer discovery via DHT.  Multiple nodes can be passed by using the flag multiple times but not comma-separated. You can also pass YAML files containing multiple nodes.",
-		Value: cli.NewStringSlice(params.BeaconNetworkConfig().BootstrapNodes...),
```

Comment:
We should instead just change to `if cliCtx.IsSet(cmd.BootstrapNode.Name) {` instead of ` len(cliCtx.StringSlice(cmd.BootstrapNode.Name)) > 0 ` now that the library is fixed

Reply:
Oh that's a good news. Than I will fix it to use `IsSet()` instead of removing default value from flag.